seed/views/v3/tax_lot_properties.py

```diff
@@ -168,18 +186,32 @@ def export(self, request):
             else:
                 view_id_str = 'taxlot_view_id'
             data.sort(key=lambda inventory_obj: order_dict[inventory_obj[view_id_str]])
-
         export_type = request.data.get('export_type', 'csv')
 
         filename = request.data.get('filename', f"ExportedData.{export_type}")
-
+        progress_data.finish_with_success()
         if export_type == "csv":
             return self._csv_response(filename, data, column_name_mappings)
         elif export_type == "geojson":
             return self._json_response(filename, data, column_name_mappings)
         elif export_type == "xlsx":
             return self._spreadsheet_response(filename, data, column_name_mappings)
 
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_member')
+    @action(detail=False, methods=['GET'])
+    def start_export(self, request):
+        """
+        Generate a ProgressData object that will be used to monitor property and tax lot exports
+        """
+        org_id = self.get_organization(request)
+
+        progress_data = ProgressData(func_name='export_inventory', unique_id=org_id)
```

Comment:
IDK if this is totally necessary, but do we need to "clear" the progress data in case a user recently exported another file? Another option could be to tweak the unique_id so that we don't end up with overlap

Reply:
Something like `progress_data.delete()`?

Reply:
Clearing the progress would prevent concurrent exports.  I've made the unique_id more unique to prevent said overlap!